kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py

```diff
@@ -387,6 +445,21 @@
                     "mozilla-account",
                 ],
             },
+            {
+                "title": "Notifications",
+                "slug": "notifications",
+                "description": "Manage notifications in Mozilla’s products.",
+                "products": [
```

Comment:
per the taxonomy map doc, the Notifications tier 2 is available only for: firefox, mobile and ios products